### DISCUSSION TITLE
Where to contribute for doctrine orm related PR

### DISCUSSION THREAD

**Discussion by Zzareb** [api-platform/core#6582] [Q&A] [answered]
Hello, 

Curently thinking about contributing but I have a question : where would I have to contribute if i wanted to contribute to the api platform doctrine orm "bundle" ? 

api platform core ? or on the api platform doctrine orm dedicated repo ?

thanks in advance.

**Comment by dunglas** [comment, marked as answer]
Thanks for contributing!! All contributions take place in the "core" monorepository.

The dedicated repositories are read-only subtree splits of the core monorepo.

**Reply by Zzareb** [reply]
Thank you.
